cloud_param: "/camera/depth/color/points"
cloud_output: "camera_depth_color_points_passthrough"
filter_field_name: "z"
filter_limit_min: 2.0
filter_limit_max: 5.0